agent/util/subtree_windows.go

```diff
@@ -138,7 +138,7 @@ func (r *processRegistry) removeJob(taskId string) error {
 // job object, which can later be used by "cleanup" to terminate all members of the job object at
 // once. If a job object doesn't already exist, it will create one automatically, scoped by the
 // task ID for which the shell process was started.
-func TrackProcess(taskId string, pid int, logger grip.Journaler) {
+func TrackProcess(pid int, taskId string, logger grip.Journaler) {
```

Comment:
Do we not need to keep track of all the processes in a list here, similar to what we do for unix?

Reply:
Ah never mind I read some more, I see this is only an issue for macOS where variable based cleanup no longer works.